src/Umbraco.Web/PublishedCache/NuCache/DataSource/DatabaseDataSourceConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Umbraco.Web.PublishedCache.NuCache.DataSource
+{
+    public class DatabaseDataSourceConfiguration
+    {
+        public DatabaseDataSourceConfiguration(bool publishedContentOnly = false)
+        {
+            PublishedContentOnly = publishedContentOnly;
+        }
+        public bool PublishedContentOnly { get; private set; }
```

Comment:
Need some code comments so future devs know what this is configuring and the outcome and prob best to be internal for now since it's only used internally. This will all have to be converted to public IOptions in netcore project.